src/index.html

```diff
@@ -2,16 +2,161 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="form__field">
+        <legend>Personal information</legend>
+        <label class="form__label">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form__label">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form__label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form__label">
+          Full date of birth:
+          <input
+            type="date"
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            autocomplete="off"
            required
          >
        </label>
        
        <label class="form__label">
          Name:
          <input
            type="text"
            name="Name"
            autocomplete="off"
            required
          >
        </label>
        
        <label class="form__label">
          How old are You?
          <input
            type="number"
            name="age"
            min="1"
            max="100"
            value="12"
          >
        </label>
        
        <label class="form__label">
          Full date of birth:
          <input
            type="date"
```


можна додати пусту строку між блоками, для кращої читабельності коду